includes/glossary.md

```diff
@@ -0,0 +1,6 @@
+*[Autoware]: OSS-based self-driving software
+*[CARET]: Chain-Aware ROS Evaluation Tool
+*[DDS]: Data Distribution System
+*[LTTng]: Linux Tracing Tool next generation
+*[OS]: Operating System
+*[ROS]: Robot Operating System \*[TILDE]: Tilde Is Latency Embedding (Integrated Latency Deadline Estimator)
```

Comment:
It appears that the line breaks are missing.
```suggestion
*[ROS]: Robot Operating System
*[TILDE]: Tilde Is Latency Embedding (Integrated Latency Deadline Estimator)
```

Reply:
@hsgwa  
> BTW, are the following lines required for all markdown files?
I found that the following link is helpful us to add abbreviations to all pages. Please wait for a while.

https://github.com/autowarefoundation/autoware-documentation/blob/cc1444548308785ec59d148521377bf45117a246/mkdocs.yaml#L79-L81